test/powershell/Modules/Microsoft.PowerShell.Utility/Import-Alias.Tests.ps1

```diff
@@ -44,40 +44,92 @@ Describe "Import-Alias DRT Unit Tests" -Tags "CI" {
 }
 
 Describe "Import-Alias" -Tags "CI" {
-	$newLine=[Environment]::NewLine
-	$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ImportAliasTestDirectory
-    $testAliases        = "pesteralias.txt"
-    $pesteraliasfile       = Join-Path -Path $testAliasDirectory -ChildPath $testAliases
 
-	BeforeEach {
-		New-Item -Path $testAliasDirectory -ItemType Directory -Force
+	BeforeAll {
+		$newLine=[Environment]::NewLine
+
+		$testAliasDirectory = Join-Path -Path $TestDrive -ChildPath ImportAliasTestDirectory
+		$aliasFilename = "pesteralias.txt"
+		$aliasFilenameMoreThanFourValues = "aliasFileMoreThanFourValues.txt"
+		$aliasFilenameLessThanFourValues = "aliasFileLessThanFourValues.txt"
+
+		$aliasfile = Join-Path -Path $testAliasDirectory -ChildPath $aliasFilename
+		$aliasPathMoreThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameMoreThanFourValues
+		$aliasPathLessThanFourValues = Join-Path -Path $testAliasDirectory -ChildPath $aliasFileNameLessThanFourValues
+
+		$commandToAlias = "echo"
+		$alias1 = "pesterecho"
+		$alias2	= '"abc""def"'
+		$alias3	= '"aaa"'
+		$alias4	= '"a,b"'
+
+		# create alias file
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
 
-		$pesteraliascontent ='# Alias File'+$newLine
-		$pesteraliascontent+='# Exported by : alex'+$newLine
-		$pesteraliascontent+='# Date/Time : Thursday, 12 November 2015 21:55:08'+$newLine
-		$pesteraliascontent+='# Computer : archvm'+$newLine+'"pesterecho","echo","","None"'
-		$pesteraliascontent > $pesteraliasfile
+		# set header
+		$aliasFileContent ='# Alias File'+$newLine
+		$aliasFileContent+='# Exported by : alex'+$newLine
+		$aliasFileContent+='# Date/Time : Thursday, 12 November 2015 21:55:08'+$newLine
+		$aliasFileContent+='# Computer : archvm'
+
+		# add various aliases
+		$aliasFileContent+= $newLine+$alias1+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias2+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias3+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent+= $newLine+$alias4+',"'+$commandToAlias+'","","None"'
+		$aliasFileContent > $aliasfile
+
+		# create invalid file with more than four values
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+		$aliasFileContent = $newLine+'"v_1","v_2","v_3","v_4","v_5"'
+		$aliasFileContent > $aliasPathMoreThanFourValues
+
+		# create invalid file with less than four values
+		New-Item -Path $testAliasDirectory -ItemType Directory -Force > $null
+		$aliasFileContent = $newLine+'"v_1","v_2","v_3"'
+		$aliasFileContent > $aliasPathLessThanFourValues
 	}
 
-	AfterEach {
+	AfterAll {
 		Remove-Item -Path $testAliasDirectory -Recurse -Force
```

Comment:
This isn't necessary as `$TestDrive` is automatically cleaned up by Pester within the scope of a `Describe` or `Context`, so if the alias directory affects individual test cases, then it should be `AfterEach`, otherwise this should be removed completely.